src/components/grid/GridLayout/GridLayout.tsx

```diff
@@ -1,10 +1,9 @@
 // Copyright 2020 Cognite AS
```

Comment:
What does GridContainer and GridLayout represent ceonceptually? Can you add comments explaining that?

Reply:
GridLayout only a wrapper component to the react-grid-layout library (leaf component). it has things like styling, managing resizing. but the grid container has all capabilities such as manage state(data), handle side effects, etc.

Reply:
please add comments in the code related to these.